### ISSUE TITLE
Training loss does not decline

### ISSUE THREAD

**Issue by xszld0v0** [shaoshitong/fractalnet#1]
Hello, I use fractalnet as a backbone for segmentation task, and I have encountered the same problem as you. During the training process, the training and validation loss doesn't decline. Have you solved this problem? 

Looking forward for your reply, thank you : )

**Comment by shaoshitong** [comment]
> Hello, I use fractalnet as a backbone for segmentation task, and I have encountered the same problem as you. During the training process, the training and validation loss doesn't decline. Have you solved this problem?
> 
> Looking forward for your reply, thank you : )

Can you provide the config of the fractalnet/task hperparameter parameters and the detailed logs? I have not used fractalnet as a backbone for semantic segmentation tasks, but I can try and analyze this.
